.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       matrix:
         django-version: [2.2, 3.1, 3.2]
         python-version: [3.6, 3.7, 3.8, 3.9]
-        elastic-version: [1.7, 2.4, 5.5]
+        elastic-version: [1.7, 2.4, 5.5, '7.13.1']
```

Comment:
I'm wondering whether we want to pin this at a point version which presumably will change regularly or can just use `7.13`